package.json

```diff
@@ -35,5 +35,21 @@
     "joi": "9.2.0",
     "sugar": "2.0.6"
   },
-  "typings": "./index.d.ts"
+  "typings": "./types",
+  "typesVersions": {
+    "*": {
```

Comment:
Usually `typesVersion` are used to support different versions of typescript. To correctly resolve it's you should export all types in `index.d.ts`?

Reply:
Yes, but that means I would modify generated files. This is the only way I could find without modifying the generated `.d.ts` files.
Got it from here: https://antfu.me/posts/types-for-sub-modules